packages/flutter_driver/test/src/real_tests/stubs/mock_callback.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:mockito/mockito.dart';
+
+class MockCallback extends Mock {
+  void call();
+}
```

Comment:
Let's avoid this - see my comment in the other file. Instead, wherever you would use this, just use a regular `VoidCallback` that simply increments a counter or sets a bool when it is invoked.